src/modules/previewpane/common/cominterop/RECT.cs

```diff
@@ -16,22 +16,22 @@ public struct RECT
         /// <summary>
         /// Specifies the x-coordinate of the upper-left corner of the rectangle.
         /// </summary>
-        public readonly int Left;
+        public int Left;
 
         /// <summary>
         /// Specifies the y-coordinate of the upper-left corner of the rectangle.
         /// </summary>
-        public readonly int Top;
+        public int Top;
 
         /// <summary>
         /// Specifies the x-coordinate of the lower-right corner of the rectangle.
         /// </summary>
-        public readonly int Right;
+        public int Right;

```

Comment:
Any reason to not make this field readonly?

Reply:
It's [used for testing](https://github.com/microsoft/PowerToys/blob/a9e2e6cc1bb614baea29246662bc770baef40b40/src/modules/previewpane/UnitTests-PreviewHandlerCommon/PreviewHandlerBaseTests.cs#L375). Not able to initialize fields for `RECT`.